codespell_lib/data/dictionary_code.txt

```diff
@@ -16,6 +16,7 @@ dateset->dataset
 datesets->datasets
 define'd->defined
 deque->dequeue
+distros->distro's
```

Comment:
Why distro's?

Reply:
same as above: spellcheck in VScode highlighted it as error and, when I used codespell in CI

Reply:
I believe the proper plural is `distros`, while `distro's` might be OK for possessive. VScode is probably wrong.

Reply:
NP - closing the PR

Reply:
Thank you for taking the time to share your findings.